libs/hpc-ui/src/lib/components/form-fields/async-autocomplete-field.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Autocomplete,
+  AutocompleteProps,
+  CircularProgress,
+} from '@mui/material';
+import { useField, useFormikContext } from 'formik';
+import { useEffect, useState } from 'react';
+import tw from 'twin.macro';
+import { FormObjectValue } from '@unocha/hpc-data';
+import { StyledTextField } from './text-field';
+
+const StyledAutocomplete = tw(Autocomplete)`
+    min-w-[10rem]
+    w-full`;
```

Comment:
Not something we can enforce, but these definitely look nicer if they are formatted like rest of the code:
```suggestion
const StyledAutocomplete = tw(Autocomplete)`
  min-w-[10rem]
  w-full
`;
```